src/core/hash.zig

```diff
@@ -10,6 +10,10 @@ pub const SlotAndHash = struct {
     slot: Slot,
     hash: Hash,
 
+    pub fn tuple(self: SlotAndHash) struct { Slot, Hash } {
```

Comment:
I have generally found the named field easier to work with than the tuple which require indices to access. hence the question: what will you say is the benefit of this?

Also it does not look like it is used anywhere.

Reply:
It actually is used, specifically in `edge_cases.processAncestorHashesDuplicateSlots`.
To answer: as demonstrated in that function it is just an easy way to have the slot hash pair named one thing, and then be able to also extract it into a slot and hash variable in one line, with the static assurance that you're extracting all the data.